arduino/libraries/RFM69_LowPowerLab/Examples/DeepSleep_usingLowPowerLibrary/DeepSleep_usingLowPowerLibrary.ino

```diff
@@ -3,7 +3,7 @@
 // Everything is put to sleep including the MCU, the radio (if any) and the FlashMem chip
 //**** SETTINGS *********************************************************************************************
 #define WITH_RFM69              //comment this line out if you don't have a RFM69 on your Moteino
-#define WITH_SPIFLASH           //comment this line out if you don't have the FLASH-MEM chip on your Moteino
```

Comment:
careful modifying 3rd party stuff. Examples are fine, but fundamental code is generally a bad idea as we might assume it's fine and then forget we did changes.

Reply:
did I use/modify that?